Corona-Warn-App/src/main/res/values/styles.xml

```diff
@@ -66,6 +66,16 @@
         <item name="android:tint">@color/colorTextPrimary1</item>
     </style>
 
+    <style name="CWAToolbar.BackArrow">
```

Comment:
Should this have  `CWAToolbar` as parent?

Reply:
yes